pkg/minikube/registry/drvs/hyperkit/hyperkit.go

```diff
@@ -85,5 +92,70 @@ func status() registry.State {
 		return registry.State{Installed: true, Error: fmt.Errorf("%s failed:\n%s", strings.Join(cmd.Args, " "), out), Fix: "Run 'brew install hyperkit'", Doc: docURL}
 	}
 
+	// Split version from v0.YYYYMMDD-HH-xxxxxxx or 0.YYYYMMDD to YYYYMMDD
+	currentVersion := convertVersionToDate(string(out))
+	specificVersion := splitHyperKitVersion(getHyperKitVersion())
+	// If current hyperkit is not newer than minimumVersion, suggest upgrade information
+	isNew, err := isNewerVersion(currentVersion, specificVersion)
+	if err != nil {
+		return registry.State{Installed: true, Healthy: true, Error: fmt.Errorf("hyperkit version check failed:\n%s", err), Doc: docURL}
```

Comment:
use %v for err, rather than %s. Don't ask me why, it's just a Go common practice. :)

Reply:
Thanks. I'll remember this manner.